jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Chip.java

```diff
@@ -31,34 +35,20 @@ public ChipAssert is() {
         return new ChipAssert().set(this);
     }
 
-    @JDIAction("Get {name}'s text")
-    public String getText() {
-        if (!find(TEXT1).getText().isEmpty()) {
-            return find(TEXT1).getText();
-        } else {
-            return find(TEXT2).getText();
-        }
-    }
-
-    @JDIAction("Get bold text from {name}'s composite label")
-    public String getBoldTextFromCompositeLabel() {
-        return find(BOLD_TEXT_FROM_COMPOSITE_LABEL).getText();
+    @Override
+    @JDIAction("Check that '{name}' is enabled")
+    public boolean isEnabled() {
+        return !hasClass("v-chip--disabled");
     }
 
-    @JDIAction("Get bold text from {name}'s composite label")
-    public String getRegularTextFromCompositeLabel() {
-        return find(REGULAR_TEXT_FROM_COMPOSITE_LABEL).getText();
+    @JDIAction("Get '{name}' content")
+    public List<UIElement> getContent() {
```

Comment:
chip content is always one, as I can see (do you have any examples for several?)
so our locator should for element ".v-chip__content" and after that it can be customized. 
It is not possible to automatically cast the list of some elements to several another classes. That's why we need one element which contains some other elements. For example for "Expandable chip" on owr site we can create a custom class with image and name

Reply:
Chip content may contain text, icons, images, and avatars. Do you mean we should create some more custom classes for all that cases?

Reply:
We can get and cast elements using stream()
I put an example in tests